### ISSUE TITLE
Random crashes that corrupt the world

### ISSUE THREAD

**Issue by SmokeyTheBandicoot** [TinyModularThings/IC2Classic#229]
Not sure what causes that, but that happens randomly (it happened 3 times now, and all 3 when I was not using IC2C). Trying to get back to world crashes the server again with the same error.

crash report: https://pastebin.com/GGLCAFEe

Edit: IC2C 1.2.1.5 MC 1.7.10

**Comment by Speiger** [comment]
@SmokeyTheBandicoot ok i checkt the code and i have 1 simple question:
This crash can only cause when you have more then 2 Trillion EU provided from a EnergySender.
IC2Classic alone can not achieve that scenario. Thats a thing of impossibility.
How did you achieve this?

**Comment by SmokeyTheBandicoot** [comment]
I did not, I have a simple solar panel (lowest tier), two coal generators and a power converter (LV) which outputs 32 EU/t from 128 RF/t. All power goes through IC2C copper cables and powers an extractor (5 overclockers), 3 singularity compressor, 1 rotary macerator. 

I noticed that before I hooked the advanced machines (singularity compressors and rotary macerator) the crash did not happen.

The crash happened when I was away from base, so I was not producing power, nor using machines

**Comment by Speiger** [comment]
From which mod is the PowerConverter ? @SmokeyTheBandicoot

**Comment by SmokeyTheBandicoot** [comment]
From the Power converter mod. 
This is what I found in mcmod.info file inside the jar, hope it can help you

[
{
  "modid": "PowerConverters",
  "name": "Power Converters",
  "description": "Take control of your power",
  "version": "1.7.10_3.2.1",
  "mcversion": "1.7.10",
  "url": "http://www.minecraftforum.net/topic/1695968-",
  "updateUrl": "",
  "authors": [
    "PowerCrystals",
    "samrg472",
    "portablejim"
  ],
  "credits": "",
  "logoFile": "",
  "screenshots": [
  ],
  "parent":"",
  "dependencies": [
  ]
}
]

Basically it is a mod which uses a block called Energy Bridger to convert one type of power (EU, RF, Charge) to another one. I got this power converter long ago the crash happened, it is just now that I got the new machines that it crashes the enet. Not sure why

It crashed again, this time I was automating botania with vanilla redstone. World corrupted again (backups, yay). Can provide log and report of course, among with world download and modpack

**Comment by Speiger** [comment]
I know powerconverters. The mod dev is on my blacklist of moddevs...

I suggest to remove that mod. IC2 Classic has EU-RF convention integrated.

**Comment by SmokeyTheBandicoot** [comment]
Well, I use that mod just to use the universal charger and connect it to a random thing player interface to charge my items remotely. If you think that Power Converters is causing that, then I'll remove and disable the IC2 LV producer. I'll see if the issue happens again.

BTW what are the conversion methods included in IC2C?

**Comment by laz2727** [comment]
Electric Flux Generators. There's no way to convert RF -> EU because it completely annihilates what little balance IC2C still has.

**Comment by SmokeyTheBandicoot** [comment]
Allright, that's fine and has sense. Guess I'll go with geothermal, then

**Comment by Speiger** [comment]
@SmokeyTheBandicoot
Since you are using 1.7.10 there are ways in ic2c to make EU out oft RF. Its Flux Generators.
They are removed in 1.10.2.
@Princess made a mistake there.

But be warned by default: 40RF => 1EU.

**Comment by Speiger** [comment]
Basically there are two types of generators.
Texture wise:

Red Dots: RF
Gray Dots: EU
3Dots: Input
1Dot: Output

You See what i mean Ehen you See the flux generators.

**Comment by SmokeyTheBandicoot** [comment]
Oh, I see... and I suppose that "small", "medium" and "big" stand for LV, MV and HV, right?

**Comment by SmokeyTheBandicoot** [comment]
Got the crash again, this time without power converters (Installed but ot using it), same crash.

**Comment by SmokeyTheBandicoot** [comment]
Any news on this? I removed power converters but the crash still happens the same

**Comment by Speiger** [comment]
well actually no. i have no idea why that happens.

**Comment by SmokeyTheBandicoot** [comment]
Could you add a check to catch the exception? Wouldn't solve the problem, but at least would stop the game to crash every other minute. Would it be hard to implement? Not sure if you're gonna push more 1.7 builds, tho

And obv thanks for your time and patience :)

**Comment by Speiger** [comment]
Closed because added to the todolist
